gcloud/monitoring/client.py

```diff
@@ -195,6 +203,121 @@ def metric_descriptor(self, type_,
             display_name=display_name,
         )
 
+    @staticmethod
```

Comment:
API consistency. Why else would the static method decorator exist in the language?

If you do:

`metric_descriptor = client.metric_descriptor()`
`query = client.query()`

seemed logical extension to do:

`metric = client.metric()`